src/Units/KaijuActions.java

```diff
@@ -214,25 +250,45 @@ public static void enqueueSingleTileEvents(MapMaster gameMap, int startingMP, Un
         {
           Environment oldEnvirons = location.getEnvironment();
           Environment newEnvirons = Environment.getTile(oldEnvirons.terrainType.getBaseTerrain(), oldEnvirons.weatherType);
-          crushEvents.add(new MapChangeEvent(location.getCoordinates(), newEnvirons));
+          crush.events.add(new MapChangeEvent(location.getCoordinates(), newEnvirons));
+
+          // Overwrite existing popup, but don't spawn one
+          if( null != tileToast )
+            tileToast = "WRECK";
```

Comment:
Why only flag terrain destruction if we are also killing a unit?

Reply:
The popup looked bad to me at the end of the movement, and it's always at the end if you're wrecking a building (unless Rampage is on).
(specifically, it looked weird 'cause we have that down-arrow thing, which shows on top of the movement arrow+menu)
![image](https://user-images.githubusercontent.com/8674241/201449276-1bc09b40-3245-42bf-804c-dacebf100e2f.png)